index.js

```diff
@@ -18,26 +18,42 @@ module.exports = fp(async function (app, { all, policy, ttl, cacheSize, skip, st
   onSkip = onSkip || noop
 
   let cache = null
+  let logTimer = null
+  let cacheReport = null
+
+  if (logInterval && policy.Query) {
+    initCacheReport()
+  }
 
   app.graphql.cache = {
     refresh () {
       buildCache()
-      setupSchema(app.graphql.schema, policy, all, cache, skip, storage, onHit, onMiss, onSkip)
+      setupSchema(app.graphql.schema, policy, all, cache, skip, storage, onHit, onMiss, onSkip, cacheReport)
     },
 
     clear () {
       cache.clear()
+      clearCacheReport()
     }
   }
 
   app.addHook('onReady', async () => {
     app.graphql.cache.refresh()
+    if (logInterval && policy.Query) {
```

Comment:
why this is not active for `all`?

Reply:
That's because I think we are not caching `mutations`. But yes `subscriptions` maybe needs to be included?

Reply:
> That's because I think we are not caching mutations. Should the subscriptions be included?
Or do you think it's better to remove this policy check

how does that impact this reporting? I don't understand.

Subscriptions should likely be skipped as well... can you send a different PR?

Reply:
My understanding was that since we are not caching `mutations` and `subscriptions` we need not include them in the cache reporting. Maybe I have misunderstood something 🤔 

Should I close this PR and send a different one with the policy check removed?

Reply:
```suggestion
    if (cacheReportingEnabled) {
```